snuba/datasets/transactions_processor.py

```diff
@@ -55,9 +55,9 @@ def __extract_timestamp(self, field: int) -> Tuple[datetime, int]:
         return (timestamp, milliseconds)
 
     def process_message(
-        self, message: Mapping[str, Any], metadata: KafkaMessageMetadata
+        self, message: Tuple[int, str, Any, Any], metadata: KafkaMessageMetadata
```

Comment:
Isn't it usually `Tuple[int, str, Any]` without the last item?

Reply:
I copied this from the events processor, which has 4. I can get rid of the last one I guess, it's not used.

Reply:
I could be wrong, I didn't not check the actual message formats we are getting. But our tests pass in 3 items here not 4.